tests/test_ble_utils_service_targets.py

```
@@ -491,6 +499,95 @@ def _reject_kwarg(address: str, **kwargs: object) -> DummyClient:
+    try:
+        minimal_handler = SimpleNamespace(
+            execute_management_command=lambda *_args, **_kwargs: "ok"
+        )
+        iface._get_management_command_handler = lambda: minimal_handler
+        resolved = BLEManagementCommandsService._handler_for_shim(iface)
+        assert resolved is minimal_handler
+        assert BLEManagementCommandsService._has_required_handler_entrypoint(
+            minimal_handler
+        )
+
+        iface._get_management_command_handler = lambda: MagicMock()
```

Ruff (0.15.6)

[warning] Lambda may be unnecessary; consider inlining inner function

Inline function call

(PLW0108)